aea/test_tools/test_cases.py

```diff
@@ -933,6 +933,49 @@ def teardown_class(cls) -> None:
         cls.agent_name = ""
 
 
+class AEATestCaseEmptyFlaky(AEATestCaseEmpty):
+    """
+    Test case for a default AEA project.
+
+    This test case will create a default AEA project.
+
+    Use for flaky tests with the flaky decorator.
+    """
+
+    failure_count: int = 0
+
+    @classmethod
+    def setup_class(cls) -> None:
+        """Set up the test class."""
+        method_list = [
+            func
+            for func in dir(cls)
+            if callable(getattr(cls, func))
+            and not func.startswith("__")
+            and func.startswith("test_")
+        ]
+        if len(method_list) > 1:
+            raise ValueError(f"{cls.__name__} can only contain one test method!")
+        super(AEATestCaseEmptyFlaky, cls).setup_class()
+        cls.failure_count = 0
```

Comment:
this could be a boolean, as we don't care how many times a test failed (in fact, we only compare if `==0`).

Reply:
You're right. I'll keep it unless other issues need changing.